basic/src/test/java/org/jboss/hal/testsuite/test/configuration/elytron/ElytronOperations.java

```diff
@@ -0,0 +1,55 @@
+package org.jboss.hal.testsuite.test.configuration.elytron;
+
+import static org.wildfly.extras.creaper.core.online.Constants.EXTENSION;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+public class ElytronOperations {
```

Comment:
Can you please add Javadoc describing this class? I know that this is common pattern, but it could be helpful for outsiders. It would be nice to have methods annotated as well but I think that their purpose will be obvious from class description.